src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -503,4 +515,10 @@ public static JsonObjectBuilder json( AuthenticationProviderRow aRow ) {
         }
         return bld;
     }
+
+    public static void addNullAware(JsonObjectBuilder builder, String name, String value)  {
```

Comment:
Is this method necessary? Can we make use of @michbarsinai 's https://github.com/IQSS/dataverse/blob/v4.3/src/main/java/edu/harvard/iq/dataverse/util/json/NullSafeJsonBuilder.java instead?